com.microsoft.AzureStorageExplorer.desktop

```diff
@@ -0,0 +1,11 @@
+[Desktop Entry]
+Type=Application
+Name=Azure Storage Explorer
+GenericName=Azure storage management
+Comment=Manage Azure cloud storage resources from your desktop
+Icon=com.microsoft.AzureStorageExplorer
+Exec=
```

Comment:
Why missing?

Reply:
According to the docs (https://docs.flatpak.org/en/latest/freedesktop-quick-reference.html) the Exec can be commited, in which case the Exec in the generated .desktop file will just not contain the --command flag.

Reply:
I know it can be empty, just not sure if it's a good idea

Reply:
I think it's best to add it even if flatpak and the command makes it redundant.

Reply:
Ok, I added it.